### ISSUE TITLE
Chrome Addon cannot load password list

### ISSUE THREAD

**Issue by gruessung** [marius-wieschollek/passwords-webextension#25]
I always get the following message. I also tried to enable legacy API.
![fehlerpwnextcloudchrome](https://user-images.githubusercontent.com/433331/43899342-e64ca808-9be1-11e8-98ab-87a6ecbfae45.png)

**Comment by marius-wieschollek** [comment]
Please check your settings in the app and also check the Nextcloud log for errors.
This error is usually caused by a 404, 403 or 500 page.

**Comment by marius-wieschollek** [comment]
You can check the Nextcloud Log in Nextcloud 12 by navigating to the Admin Area and checking the Log Section. 

The extension can be debugged as [described here](https://github.com/marius-wieschollek/passwords-webextension/blob/master/Contributing.md#debugging).

**Comment by boostedgti916** [comment]
I am getting a net::ERR_TOO_MANY_REDIRECTS error?

**Comment by CleanaZ** [comment]
I got the same problem. I made a screenshot of chrome's console output.

![image](https://user-images.githubusercontent.com/1440458/44388943-c9a3d680-a529-11e8-8bd0-dad53ff57355.png)

I have checked the directory structure with an ftp program and i cannot see any folder named 
.../apps/passwords/**api**/**0.1**/..

These are the available folders at /apps/passwords/ 
![image](https://user-images.githubusercontent.com/1440458/44389138-58b0ee80-a52a-11e8-9b68-11873dacf27b.png)

Is the chrome extension maybe using a wrong path ?

**Comment by marius-wieschollek** [comment]
The chrome extension is using the correct path. You need to configure your webserver to resolve these virtual paths correctly. Take a look at the [Nextcloud Documentation](https://docs.nextcloud.com/server/13/admin_manual/installation/index.html) and also fix all the error messages shown in the basic settings section in the admin settings.

**Comment by CleanaZ** [comment]
I am wondering why the webserver should not be able to resolve the paths because i can fully use your app within nextcloud. Only with the chrome extension it's not possible to store or show the passwords.
And it's not possible for me to change anything at the server configuration because i am using a webosting package ([PrivatePlus](https://all-inkl.com/webhosting/privatplus/)) from All-Inkl.com - with these hosting packages it's possible to (let) install different software like NextCloud/OwnCloud by an automated software installer from all-inkl.com 
And nowadays most of the webhosting providers are offering such [(automated) software installer](https://all-inkl.com/webhosting/software-installer/), but without any possibility to change server settings (so there will be a lot of other users with the same problems). I had made a phone call with the support of all-inkl.com and told them about the needed improvements as shown at the basic settings section (look at the following screenshot). But they told me that the will not change anything because nextcloud is also working without that improvements.
![image](https://user-images.githubusercontent.com/1440458/44431179-5a67ca00-a59d-11e8-8a50-dab0636f2ea7.png)

Is there no other way to fetch the password-data from my nextcloud to the chrome-extension [maybe using a path setting in one of the config(.php) files of nextcloud thus it will read the real path out of a file and work without such virtual paths? Or can i write any command to a htaccess file within nextcloud to make it work? - i cannot do any configuration at the server but i can edit all files on my webspace)

**Comment by marius-wieschollek** [comment]
@CleanaZ The admin page from your Nextcloud looks ok to me, only warnings in red need to be fixed.

I did some debugging on the extension over the weekend and i think that there is indeed an incompatibility with newer versions of Chromium based browsers. 
There was a similar issue with Firefox 61 a few weeks back caused by a change in the [default browser settings for the fetch api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch). Chromium did now introduce the same change. 
Before, cookies were not sent to the server when the fetch api was used - now they are. This means that now by default your Nextcloud session is used to access the external Nextcloud api, which Nextcloud does not allow. That seems to cause a redirect loop.

I originally implemented and shipped a fix for Firefox only because Chromium was not affected at that time. I have now released [version 1.5.2](https://github.com/marius-wieschollek/passwords-webextension/wiki/chromium-builds) for Chromium and hope that this fixes the issue for you.

Again: There is most probably nothing wrong with your Nextcloud instance.
